Description: Restores deprecated Qt constant in order to compile on stable
 .
 libtdesktopenvironment (0.2-1) UNRELEASED; urgency=low
 .
   * Initial release.
 .
   * Known bugs: libkf5pulseaudioqt only available on unstable, so support
                 for it is unavailable for now.
Author: Zumi Daxuya <daxuya.zumi+code@protonmail.com>

---
Last-Update: 2020-07-21

--- libtdesktopenvironment-0.2.orig/lib/Applications/application.cpp
+++ libtdesktopenvironment-0.2/lib/Applications/application.cpp
@@ -158,7 +158,7 @@ QStringList Application::getStringList(Q
     if (!d->isValid) return QStringList();
 
     QString property = getProperty(propertyName, defaultValue).toString();
-    return property.split(";", Qt::SkipEmptyParts);
+    return property.split(";", QString::SkipEmptyParts);
 }
 
 QStringList Application::allApplications(QStringList searchPaths) {
